vsphere/datadog_checks/vsphere/metrics.py

```diff
@@ -10,39 +10,21 @@
 PERCENT_METRICS = {
     'cpu.capacity.contention.avg',
     'cpu.coreUtilization.avg',
-    'cpu.coreUtilization.max',
-    'cpu.coreUtilization.min',
-    'cpu.coreUtilization.raw',
     'cpu.corecount.contention.avg',
```

Comment:
Since we have only one datapoint, IMO `.avg/.sum` suffixes might be confusing for users.

I would favour dropping it :) Is there any conflicting cases if we drop it? I mean, having 2 metrics with same name.

>  code-change would have been much bigger.

Agree, but should not be hard I think.

Reply:
I'm in favor of keeping it :) 
That way the updated ootb dashboard can be compatible with both implementation, and that makes it clear what has been requested.

`.avg` and `.sum` are helpful to understand how to understand the metric, in case you want to set up manual rollups operations.

Reply:
> Is there any conflicting cases if we drop it?
No conflicting case no

Reply:
Fair enough, let's keep it then :)